### PR TITLE
Feature: PHeading component

### PR DESCRIPTION
**Pull request by pleek91**
# Description
A simple heading component
image

### CODE REVIEW

File: src/components/HashLink/PHashLink.vue [hunk 1]
```diff
@@ -40,6 +40,7 @@
   cursor-pointer
   relative
   transition-all
+  flex
```
Comment (znicholasbrown) [inline src/components/HashLink/PHashLink.vue:43]: What's the reason for this change?

Comment (pleek91) [reply src/components/HashLink/PHashLink.vue:43]: So you can put a `p-heading` inside of a hashlink. Otherwise the hash and the text are on separate lines. This going to cause issues? I can revert. Only reason I added this was to use the p-heading component on the demo page

Comment (znicholasbrown) [reply src/components/HashLink/PHashLink.vue:43]: Ahh - this may need to be updated in a few places then, the change breaks this section of the markdown renderer and probably elsewhere that I haven't thought to check:
Screen Shot 2023-03-08 at 12 15 20 PM

**Review by znicholasbrown — APPROVED** [review]

File: src/components/HashLink/PHashLink.vue (hunk 1 above)
Comment (pleek91) [reply src/components/HashLink/PHashLink.vue:43]: ahh good catch. Pushed a fix for that in the markdown parser. I cannot find any other uses of PHashLink outside of the demo in any of our projects.

**Comment by brandonreid** [comment]
Nice!
